project/templates/unicoremc/home.html

```diff
@@ -17,22 +17,25 @@
             <thead>
                 <tr>
                     <th scope="col">
-                        <div class="grp-text">&nbsp;</div>
+                        <div class="grp-text">App Type</div>
                     </th>
                     <th scope="col">
-                        <div class="grp-text">&nbsp;</div>
+                        <div class="grp-text">Country</div>
                     </th>
                     <th scope="col">
-                        <div class="grp-text">&nbsp;</div>
+                        <div class="grp-text">Status</div>
                     </th>
                     <th scope="col">
                         <div class="grp-text">Content Store</div>
                     </th>
                     <th scope="col">
-                        <div class="grp-text">Frontend URL</div>
+                        <div class="grp-text">Frontend URL <br />*.hub.unicore.io</div>
```

Comment:
nah, QA MC is hardly used by anyone apart from the dev team since we use `00.*.hub.unicore.io` for training. I'm happy to leave it slightly ambiguous for qa. The links will still say `za.ffl.qa-`.